utils/publish.mjs

```diff
@@ -18,16 +18,20 @@ export const checkTag = (tag) => {
     return true;
 };
 
-export const publishDirectoryAsTag = (distPath, origin, tag, dryRun) => {
+export const publishDirectoryAsTags = (distPath, origin, tag, majorTag, dryRun) => {
     const cmds = [
         `git init .`,
         `git add .`,
         `git commit -m publish`,
         `git remote add origin ${origin}`,
         `git tag ${tag}`,
+        `git tag ${majorTag}`,
     ];
     if (!dryRun) {
         cmds.push(`git push origin ${tag}`);
+        // This will succeed with a warning if the major tag doesn't exist
+        cmds.push(`git push origin :refs/tags/${majorTag}`);
+        cmds.push(`git push origin ${majorTag}`);
```

Comment:
So this overwrite the major tag with whatever the most updated to minor.patch version within that major version is?  Nice solution.